src/IoTeX/Staking.h

```diff
@@ -0,0 +1,43 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "Data.h"
+
+namespace TW::IoTeX {
+
+static const char *_Nonnull STAKING_CONTRACT = "io1xpq62aw85uqzrccg9y5hnryv8ld2nkpycc3gza";
+
+Data Stake(
+    const byte _canName[12],
+    const uint64_t _stakeDuration,
+    const bool _nonDecay,
+    uint64_t dataLen,
+    const byte *_data);
+
+Data Unstake(
+    uint64_t _pyggIndex,
+    uint64_t dataLen,
+    const byte *_data);
```

Comment:
These function declarations are not very long, suggest arranging them in one line, at least here in the .h file.

Reply:
ok